administrator/components/com_installer/Model/UpdateModel.php

```diff
@@ -397,6 +397,9 @@ public function update($uids, $minimum_stability = \JUpdater::STABILITY_STABLE)
 	 */
 	private function install($update)
 	{
+		// Load overrides plugin.
+		PluginHelper::importPlugin('overrides');
```

Comment:
Same here: If we use the plugin group installer, as suggested, we do not need to insert anything here.